bin/circleci_promotebuild.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -x
+# Edited this script from the violetatrium/embedded-tools repo. Removed any non-required logic.
```

Comment:
@brettjaynes this is fine for now but long term we'll submodule the embedded-tools repo and make the script in there do what we want.

Reply:
That's reasonable. There were enough tweaks I needed to make to work around CircleCI (especially how its filesystem works between jobs), as well as updating the server connection details, that in the moment it seemed simpler to just take the script and rebuild it here with the modifications I need.